assets/node.yaml

```diff
@@ -24,27 +24,6 @@ spec:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
     spec:
-      initContainers:
-        - name: vpc-node-label-updater
-          securityContext:
-            privileged: false
-            allowPrivilegeEscalation: false
-          resources:
-            requests:
-              cpu: 3m
-              memory: 6Mi
-          env:
-            - name: NODE_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: spec.nodeName
-            - name: SECRET_CONFIG_PATH
-              value: /etc/storage_ibmc
-          image: ${NODE_LABEL_IMAGE}
-          imagePullPolicy: IfNotPresent
-          volumeMounts:
-            - mountPath: /etc/storage_ibmc
-              name: customer-auth
```

Comment:
@GunaKKIBM I'm happy to see the node labeler is no longer required, but I'd like to understand why.
Were there changes made to the driver so that this is no longer needed?
Is there a minimum version of the driver that we should be deploying before removing this from the node daemonset?

Reply:
Thank you @dobsonj. To answer your question - https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/commit/cfb1567c7105c8f304b3ea48329884db012c60bf#diff-b63b75282ebd8dd9151dd9952ad690f73af44f7f36d28fae186e0aa32e37515aR94 - here are the changes. VPC node label updater was earlier fetching instanceID from RIAAS, now we are reading directly from node spec. Also, this code was checked in long ago, I see that the same is reflected in https://github.com/openshift/ibm-vpc-block-csi-driver repository. Any version from 5.1.x should be good.

Reply:
https://github.com/openshift/ibm-vpc-block-csi-driver/commit/cfb1567c7105c8f304b3ea48329884db012c60bf#diff-b63b75282ebd8dd9151dd9952ad690f73af44f7f36d28fae186e0aa32e37515a

Reply:
Great, thanks very much!